diff -ruEZB avr-tinyuart-ORIGINAL/tinyuart.c avr-tinyuart/tinyuart.c
--- avr-tinyuart-ORIGINAL/tinyuart.c	2021-03-23 04:47:26.000000000 +0700
+++ avr-tinyuart/tinyuart.c	2024-03-29 05:25:10.000000000 +0700
@@ -6,16 +6,7 @@
  */ 
 
 
-#include "tinyuart.h"
-
-#if __has_include("config/tinyuart_config.h")
-	#include "config/tinyuart_config.h"
-#else
-	#include "tinyuart_config.h"
-	#pragma message "tinyuart: build-in configuration loaded, for custom settings see note tinyuart_config.h"
-#endif
-
-
+#include "tinyuart_config.h"
 
 
 //////////////////////////////////////////////////////////////////////////
@@ -161,6 +152,11 @@
 			"dec	%[cnt]						\n\t"
 			"brne	send_bit					\n\t"
 		
+			"mov	__tmp_reg__,	%[loops]	\n\t"	// stop bit delay loop
+		"stop_bit_delay:"
+			"dec	__tmp_reg__					\n\t"
+			"brne	stop_bit_delay				\n\t"
+
 		// output values, can be safely written to. "+" attribute is read-write
 		: [data]	"+r"(data),							// use original value and allow overwrite
 		  [cnt]		"=&d"(cnt)							// let compiler pick temporary register
@@ -168,7 +164,7 @@
 		// input values; never write to any of the below
 		: [mask]	"r"(mask),							// let compiler put mask in register
 		  [loops]	"r"((uint8_t)(LOOP_DELAY+1)),		// the 0th loop iteration already decrements, +1 as compensation
-		  [pin]		"I"(_SFR_IO_ADDR(_io_ddr(TINYUART_IO_TX)))
+		  [pin]		"I"(_SFR_IO_ADDR(_io_pin(TINYUART_IO_TX)))
 		
 		// cobblers: registers modified without compiler already knowing
 		: /*none*/
diff -ruEZB avr-tinyuart-ORIGINAL/tinyuart_config.h avr-tinyuart/tinyuart_config.h
--- avr-tinyuart-ORIGINAL/tinyuart_config.h	2021-03-23 04:47:26.000000000 +0700
+++ avr-tinyuart/tinyuart_config.h	2024-03-29 05:10:26.000000000 +0700
@@ -27,13 +27,13 @@
 
 // specify real frequency of the CPU. If communication is unsuccessful, you can tweak this value.
 // Beware, you CAN NOT use floating point calculations here!
-#define TINYUART_F_CPU				9600000
+#define TINYUART_F_CPU				AVR_TINYUART_F_CPU
 
 // define a baud rate
-#define TINYUART_BAUD				115200
+#define TINYUART_BAUD				AVR_TINYUART_BAUD
 
 // define IO
-#define TINYUART_IO_TX				B,3
+#define TINYUART_IO_TX				AVR_TINYUART_IO_TX
 
 
 // Options. Comment line to disable.
