diff --git a/src/com/intellectualsites/http/ClientSettings.java b/src/com/intellectualsites/http/ClientSettings.java
index a13f9a2..a8877d9 100644
--- a/src/com/intellectualsites/http/ClientSettings.java
+++ b/src/com/intellectualsites/http/ClientSettings.java
@@ -1,3 +1,7 @@
+/*
+  * #### This file has been modified by JxMake project #####
+  */
+
 /*
  * MIT License
  *
@@ -29,8 +33,8 @@
 import java.util.Objects;
 import java.util.function.Consumer;
 
-/**
- * Settings that change the behaviour of {@link HttpClient}
+/*
+ Settings that change the behaviour of {@link HttpClient}
  */
 final class ClientSettings {
 
@@ -42,60 +46,60 @@ final class ClientSettings {
         this.baseURL = "";
     }
 
-    /**
-     * Base URL that is prepended to the URL
-     * of each request
-     *
-     * @return Base URL
+    /*
+     Base URL that is prepended to the URL
+     of each request
+     
+     @return Base URL
      */
     String getBaseURL() {
         return this.baseURL;
     }
 
-    /**
-     * Get the entity mapper that should be used
-     * in all request (by default)
-     *
-     * @return Entity mapper
+    /*
+     Get the entity mapper that should be used
+     in all request (by default)
+     
+     @return Entity mapper
      */
     EntityMapper getEntityMapper() {
         return this.entityMapper;
     }
 
-    /**
-     * Get all registered request decorators
-     *
-     * @return Unmodifiable collection of decorators
+    /*
+     Get all registered request decorators
+     
+     @return Unmodifiable collection of decorators
      */
     Collection<Consumer<HttpClient.WrappedRequestBuilder>> getRequestDecorators() {
         return Collections.unmodifiableCollection(this.decorators);
     }
 
-    /**
-     * Set the base URL, that is prepended to
-     * the URL of each request
-     *
-     * @param baseURL base URL
+    /*
+     Set the base URL, that is prepended to
+     the URL of each request
+     
+     @param baseURL base URL
      */
     void setBaseURL(final String baseURL) {
         this.baseURL = Objects.requireNonNull(baseURL, "Base URL may not be null");
     }
 
-    /**
-     * Set the default entity mapper that is used
-     * by all requests, unless otherwise specified
-     *
-     * @param entityMapper Entity mapper
+    /*
+     Set the default entity mapper that is used
+     by all requests, unless otherwise specified
+     
+     @param entityMapper Entity mapper
      */
     void setEntityMapper(final EntityMapper entityMapper) {
         this.entityMapper = entityMapper;
     }
 
-    /**
-     * Add a new request decorator. This will have the opportunity
-     * to decorate every request made by this client
-     *
-     * @param decorator Decorator
+    /*
+     Add a new request decorator. This will have the opportunity
+     to decorate every request made by this client
+     
+     @param decorator Decorator
      */
     void addDecorator(final Consumer<HttpClient.WrappedRequestBuilder> decorator) {
         this.decorators.add(Objects.requireNonNull(decorator, "Decorator may not be null"));
diff --git a/src/com/intellectualsites/http/ContentType.java b/src/com/intellectualsites/http/ContentType.java
index 0d18a7c..ff838a0 100644
--- a/src/com/intellectualsites/http/ContentType.java
+++ b/src/com/intellectualsites/http/ContentType.java
@@ -1,3 +1,7 @@
+/*
+  * #### This file has been modified by JxMake project #####
+  */
+
 /*
  * MIT License
  *
@@ -27,8 +31,8 @@
 import java.util.Map;
 import java.util.Objects;
 
-/**
- * Wrapper for Content-Type header values
+/*
+ Wrapper for Content-Type header values
  */
 public final class ContentType {
 
@@ -45,11 +49,11 @@ private ContentType(final String type) {
         this.type = type;
     }
 
-    /**
-     * Get the MIME type instance corresponding to the IANA template name
-     *
-     * @param type MIME type
-     * @return MIME type instance
+    /*
+     Get the MIME type instance corresponding to the IANA template name
+     
+     @param type MIME type
+     @return MIME type instance
      */
     public static ContentType of(final String type) {
         return internalMap.computeIfAbsent(Objects.requireNonNull(type,
diff --git a/src/com/intellectualsites/http/EntityMapper.java b/src/com/intellectualsites/http/EntityMapper.java
index 9845143..3be6e78 100644
--- a/src/com/intellectualsites/http/EntityMapper.java
+++ b/src/com/intellectualsites/http/EntityMapper.java
@@ -1,3 +1,7 @@
+/*
+  * #### This file has been modified by JxMake project #####
+  */
+
 /*
  * MIT License
  *
@@ -30,8 +34,8 @@
 import java.util.Objects;
 import java.util.Optional;
 
-/**
- * Utility responsible for serializing and de-serializing HTTP entities
+/*
+ Utility responsible for serializing and de-serializing HTTP entities
  */
 public final class EntityMapper {
 
@@ -43,10 +47,10 @@ private static <T> T castUnsafe(final Object o) {
         return (T) o;
     }
 
-    /**
-     * Create a new entity mapper instance
-     *
-     * @return Created instance
+    /*
+     Create a new entity mapper instance
+     
+     @return Created instance
      */
     public static EntityMapper newInstance() {
         final EntityMapper mapper = new EntityMapper();
@@ -55,13 +59,13 @@ public static EntityMapper newInstance() {
         return mapper;
     }
 
-    /**
-     * Register a serializer that maps a given type to an array of bytes
-     *
-     * @param clazz      Class of type to map
-     * @param serializer Serializer that performs the mapping
-     * @param <T>        Type to map
-     * @return Mapper instance
+    /*
+     Register a serializer that maps a given type to an array of bytes
+     
+     @param clazz      Class of type to map
+     @param serializer Serializer that performs the mapping
+     @param <T>        Type to map
+     @return Mapper instance
      */
     public <T> EntityMapper registerSerializer(final Class<T> clazz,
         final EntitySerializer<T> serializer) {
@@ -71,13 +75,13 @@ public <T> EntityMapper registerSerializer(final Class<T> clazz,
         return this;
     }
 
-    /**
-     * Register a deserializer that maps objects of a certain content type to Java objects
-     *
-     * @param clazz        Content type
-     * @param deserializer Deserializer
-     * @param <T>          Type of the objects produces by the deserializer
-     * @return Mapper instance
+    /*
+     Register a deserializer that maps objects of a certain content type to Java objects
+     
+     @param clazz        Content type
+     @param deserializer Deserializer
+     @param <T>          Type of the objects produces by the deserializer
+     @return Mapper instance
      */
     public <T> EntityMapper registerDeserializer(final Class<T> clazz,
         final EntityDeserializer<T> deserializer) {
@@ -87,12 +91,12 @@ public <T> EntityMapper registerDeserializer(final Class<T> clazz,
         return this;
     }
 
-    /**
-     * Attempt to retrieve the serializer for a given type
-     *
-     * @param clazz Class
-     * @param <T>   Type
-     * @return Serializer
+    /*
+     Attempt to retrieve the serializer for a given type
+     
+     @param clazz Class
+     @param <T>   Type
+     @return Serializer
      */
     public <T> Optional<EntitySerializer<T>> getSerializer(final Class<T> clazz) {
         final EntitySerializer<?> serializer = this.serializers.get(clazz);
@@ -102,12 +106,12 @@ public <T> Optional<EntitySerializer<T>> getSerializer(final Class<T> clazz) {
         return Optional.of(castUnsafe(serializer));
     }
 
-    /**
-     * Attempt to retrieve the deserializer for a given content type
-     *
-     * @param type Content class
-     * @param <T>  Content type
-     * @return Deserializer
+    /*
+     Attempt to retrieve the deserializer for a given content type
+     
+     @param type Content class
+     @param <T>  Content type
+     @return Deserializer
      */
     @SuppressWarnings("unchecked")
     public <T> Optional<EntityDeserializer<T>> getDeserializer(final Class<T> type) {
@@ -119,46 +123,46 @@ public <T> Optional<EntityDeserializer<T>> getDeserializer(final Class<T> type)
     }
 
 
-    /**
-     * Serializer for HTTP request bodies
-     *
-     * @param <T> Object type
+    /*
+     Serializer for HTTP request bodies
+     
+     @param <T> Object type
      */
     public interface EntitySerializer<T> {
 
-        /**
-         * Serialize the input into a byte array, which can then be
-         * written to the HTTP request
-         *
-         * @param input Input that should be serialized
-         * @return The serialized object
+        /*
+         Serialize the input into a byte array, which can then be
+         written to the HTTP request
+         
+         @param input Input that should be serialized
+         @return The serialized object
          */
         byte[] serialize(final T input);
 
-        /**
-         * Get the content type of the object
-         *
-         * @return Content Type
+        /*
+         Get the content type of the object
+         
+         @return Content Type
          */
         ContentType getContentType();
 
     }
 
 
-    /**
-     * Deserializer for HTTP response bodies
-     *
-     * @param <T> Object type
+    /*
+     Deserializer for HTTP response bodies
+     
+     @param <T> Object type
      */
     @FunctionalInterface
     public interface EntityDeserializer<T> {
 
-        /**
-         * Deserialize the input byte array into an object.
-         *
-         * @param contentType Optional content type, if supplied by the server
-         * @param input       Input that should be de-serialized
-         * @return De-serialized input
+        /*
+         Deserialize the input byte array into an object.
+         
+         @param contentType Optional content type, if supplied by the server
+         @param input       Input that should be de-serialized
+         @return De-serialized input
          */
         T deserialize(final ContentType contentType,
             final byte[] input);
diff --git a/src/com/intellectualsites/http/Headers.java b/src/com/intellectualsites/http/Headers.java
index 06b7c2d..36d0ff4 100644
--- a/src/com/intellectualsites/http/Headers.java
+++ b/src/com/intellectualsites/http/Headers.java
@@ -1,3 +1,7 @@
+/*
+  * #### This file has been modified by JxMake project #####
+  */
+
 /*
  * MIT License
  *
@@ -31,8 +35,8 @@
 import java.util.Map;
 import java.util.Objects;
 
-/**
- * Container for HTTP headers
+/*
+ Container for HTTP headers
  */
 final public class Headers {
 
@@ -41,20 +45,20 @@ final public class Headers {
     private Headers() {
     }
 
-    /**
-     * Return an empty {@link Headers} instance
-     *
-     * @return Headers instance
+    /*
+     Return an empty {@link Headers} instance
+     
+     @return Headers instance
      */
     static Headers newInstance() {
         return new Headers();
     }
 
-    /**
-     * Add a header to the header collection
-     *
-     * @param key   Header name
-     * @param value Header value
+    /*
+     Add a header to the header collection
+     
+     @param key   Header name
+     @param value Header value
      */
     void addHeader(String key, final String value) {
         Objects.requireNonNull(key, "Key may not be null");
@@ -69,11 +73,11 @@ void addHeader(String key, final String value) {
         }
     }
 
-    /**
-     * Get a list of all the headers with the specified name
-     *
-     * @param key Header key
-     * @return Unmodifiable list
+    /*
+     Get a list of all the headers with the specified name
+     
+     @param key Header key
+     @return Unmodifiable list
      */
     public List<String> getHeaders(final String key) {
         Objects.requireNonNull(key, "Key may not be null");
@@ -84,26 +88,26 @@ public List<String> getHeaders(final String key) {
         return Collections.unmodifiableList(headers);
     }
 
-    /**
-     * Get the value of a specific header, or an empty string.
-     * If multiple values are specified for the header key,
-     * only the last value will be returned.
-     *
-     * @param key Header key
-     * @return Header value, or {@code ""}
+    /*
+     Get the value of a specific header, or an empty string.
+     If multiple values are specified for the header key,
+     only the last value will be returned.
+     
+     @param key Header key
+     @return Header value, or {@code ""}
      */
     public String getHeader(final String key) {
         return Objects.requireNonNull(getOrDefault(key, ""));
     }
 
-    /**
-     * Get the value of a specific header, or default string.
-     * If multiple values are specified for the header key,
-     * only the last value will be returned.
-     *
-     * @param key           Header key
-     * @param defaultString Default value
-     * @return Header value, or the default value
+    /*
+     Get the value of a specific header, or default string.
+     If multiple values are specified for the header key,
+     only the last value will be returned.
+     
+     @param key           Header key
+     @param defaultString Default value
+     @return Header value, or the default value
      */
     String getOrDefault(final String key, final String defaultString) {
         final List<String> headers = this.getHeaders(key);
@@ -113,10 +117,10 @@ String getOrDefault(final String key, final String defaultString) {
         return headers.get(headers.size() - 1);
     }
 
-    /**
-     * Get the name of all headers in the collection
-     *
-     * @return Unmodifiable collection
+    /*
+     Get the name of all headers in the collection
+     
+     @return Unmodifiable collection
      */
     public Collection<String> getHeaders() {
         return Collections.unmodifiableSet(this.headers.keySet());
diff --git a/src/com/intellectualsites/http/HttpClient.java b/src/com/intellectualsites/http/HttpClient.java
index cfbe737..b2f588b 100644
--- a/src/com/intellectualsites/http/HttpClient.java
+++ b/src/com/intellectualsites/http/HttpClient.java
@@ -1,3 +1,7 @@
+/*
+  * #### This file has been modified by JxMake project #####
+  */
+
 /*
  * MIT License
  *
@@ -31,8 +35,8 @@
 import java.util.function.Consumer;
 import java.util.function.Supplier;
 
-/**
- * A simple Java HTTP client
+/*
+ A simple Java HTTP client
  */
 public final class HttpClient {
 
@@ -43,94 +47,94 @@ private HttpClient(final ClientSettings settings) {
         this.settings = Objects.requireNonNull(settings);
     }
 
-    /**
-     * Create a new {@link Builder}
-     *
-     * @return Builder instance
+    /*
+     Create a new {@link Builder}
+     
+     @return Builder instance
      */
     public static Builder newBuilder() {
         return new Builder();
     }
 
-    /**
-     * Initialise a request builder for a GET request
-     *
-     * @param url URL
-     * @return Created builder
+    /*
+     Initialise a request builder for a GET request
+     
+     @param url URL
+     @return Created builder
      */
     public WrappedRequestBuilder get(final String url) {
         Objects.requireNonNull(url, "URL may not be null");
         return new WrappedRequestBuilder(HttpMethod.GET, url);
     }
 
-    /**
-     * Initialise a request builder for a POST request
-     *
-     * @param url URL
-     * @return Created builder
+    /*
+     Initialise a request builder for a POST request
+     
+     @param url URL
+     @return Created builder
      */
     public WrappedRequestBuilder post(final String url) {
         Objects.requireNonNull(url, "URL may not be null");
         return new WrappedRequestBuilder(HttpMethod.POST, url);
     }
 
-    /**
-     * Initialise a request builder for a PUT request
-     *
-     * @param url URL
-     * @return Created builder
+    /*
+     Initialise a request builder for a PUT request
+     
+     @param url URL
+     @return Created builder
      */
     public WrappedRequestBuilder put(final String url) {
         Objects.requireNonNull(url, "URL may not be null");
         return new WrappedRequestBuilder(HttpMethod.PUT, url);
     }
 
-    /**
-     * Initialise a request builder for a HEAD request
-     *
-     * @param url URL
-     * @return Created builder
+    /*
+     Initialise a request builder for a HEAD request
+     
+     @param url URL
+     @return Created builder
      */
     public WrappedRequestBuilder head(final String url) {
         Objects.requireNonNull(url, "URL may not be null");
         return new WrappedRequestBuilder(HttpMethod.HEAD, url);
     }
 
-    /**
-     * Initialise a request builder for a DELETE request
-     *
-     * @param url URL
-     * @return Created builder
+    /*
+     Initialise a request builder for a DELETE request
+     
+     @param url URL
+     @return Created builder
      */
     public WrappedRequestBuilder delete(final String url) {
         Objects.requireNonNull(url, "URL may not be null");
         return new WrappedRequestBuilder(HttpMethod.DELETE, url);
     }
 
-    /**
-     * Initialise a request builder for a PATCH request
-     *
-     * @param url URL
-     * @return Created builder
+    /*
+     Initialise a request builder for a PATCH request
+     
+     @param url URL
+     @return Created builder
      */
     public WrappedRequestBuilder patch(final String url) {
         Objects.requireNonNull(url, "URL may not be null");
         return new WrappedRequestBuilder(HttpMethod.PATCH, url);
     }
 
-    /**
-     * Get the entity mapper used by the client
-     *
-     * @return Entity mapper
+    /*
+     Get the entity mapper used by the client
+     
+     @return Entity mapper
      */
     public EntityMapper getMapper() {
         return this.mapper;
     }
 
 
-    /**
-     * Builder for {@link HttpClient}. Use {@link #newBuilder()} to create
-     * an instance of the builder
+    /*
+     Builder for {@link HttpClient}. Use {@link #newBuilder()} to create
+     an instance of the builder
      */
     public static final class Builder {
 
@@ -140,12 +144,12 @@ private Builder() {
             this.settings = new ClientSettings();
         }
 
-        /**
-         * Set the base URL. This will be prepended to each request made with the
-         * client.
-         *
-         * @param baseURL Base URL. Cannot be null, but can be empty.
-         * @return Builder instance
+        /*
+         Set the base URL. This will be prepended to each request made with the
+         client.
+         
+         @param baseURL Base URL. Cannot be null, but can be empty.
+         @return Builder instance
          */
         public Builder withBaseURL(final String baseURL) {
             Objects.requireNonNull(baseURL, "Base URL may not be null");
@@ -157,35 +161,35 @@ public Builder withBaseURL(final String baseURL) {
             return this;
         }
 
-        /**
-         * Set the default entity mapper that is used
-         * by all requests, unless otherwise specified
-         *
-         * @param entityMapper Entity mapper
-         * @return Builder instance
+        /*
+         Set the default entity mapper that is used
+         by all requests, unless otherwise specified
+         
+         @param entityMapper Entity mapper
+         @return Builder instance
          */
         public Builder withEntityMapper(final EntityMapper entityMapper) {
             this.settings.setEntityMapper(entityMapper);
             return this;
         }
 
-        /**
-         * Add a new request decorator. This will have the opportunity
-         * to decorate every request made by this client
-         *
-         * @param decorator Decorator
-         * @return Builder instance
+        /*
+         Add a new request decorator. This will have the opportunity
+         to decorate every request made by this client
+         
+         @param decorator Decorator
+         @return Builder instance
          */
         public Builder withDecorator(final Consumer<WrappedRequestBuilder> decorator) {
             this.settings.addDecorator(Objects.requireNonNull(decorator, "Decorator may not be null"));
             return this;
         }
 
-        /**
-         * Create a new {@link HttpClient} using the
-         * settings specified in the builder
-         *
-         * @return Created client
+        /*
+         Create a new {@link HttpClient} using the
+         settings specified in the builder
+         
+         @return Created client
          */
         public HttpClient build() {
             return new HttpClient(settings);
@@ -194,15 +198,15 @@ public HttpClient build() {
     }
 
 
-    /**
-     * Wrapper used to interact with HTTP requests.
-     *
-     * @see #get(String) To create a new GET request
-     * @see #post(String) To create a new POST request
-     * @see #head(String) To create a new HEAD request
-     * @see #put(String) To create a new PUT request
-     * @see #patch(String) To create a new PATCH request
-     * @see #delete(String) To create a new DELETE request
+    /*
+     Wrapper used to interact with HTTP requests.
+     
+     @see #get(String) To create a new GET request
+     @see #post(String) To create a new POST request
+     @see #head(String) To create a new HEAD request
+     @see #put(String) To create a new PUT request
+     @see #patch(String) To create a new PATCH request
+     @see #delete(String) To create a new DELETE request
      */
     public final class WrappedRequestBuilder {
 
@@ -234,36 +238,37 @@ private WrappedRequestBuilder(final HttpMethod method, String url) {
             }
         }
 
-        /**
-         * Specify an input supplier which will be used to write to the connection, if it
-         * established correctly. This requires that there is a {@link com.intellectualsites.http.EntityMapper.EntitySerializer}
-         * registered for the type of the object, in the {@link EntityMapper} used by the client
-         *
-         * @param input Input
-         * @return Builder instance
+        /*
+         Specify an input supplier which will be used to write to the connection, if it
+         established correctly. This requires that there is a
+         {@link com.intellectualsites.http.EntityMapper.EntitySerializer}
+         registered for the type of the object, in the {@link EntityMapper} used by the client
+         
+         @param input Input
+         @return Builder instance
          */
         public WrappedRequestBuilder withInput(final Supplier<Object> input) {
             builder.withInput(input);
             return this;
         }
 
-        /**
-         * Specify the entity mapper used by the request
-         *
-         * @param mapper Entity mapper
-         * @return Builder instance
+        /*
+         Specify the entity mapper used by the request
+         
+         @param mapper Entity mapper
+         @return Builder instance
          */
         public WrappedRequestBuilder withMapper(final EntityMapper mapper) {
             builder.withMapper(mapper);
             return this;
         }
 
-        /**
-         * Add a header to the request
-         *
-         * @param key   Header key
-         * @param value Header value
-         * @return Builder instance
+        /*
+         Add a header to the request
+         
+         @param key   Header key
+         @param value Header value
+         @return Builder instance
          */
         public WrappedRequestBuilder withHeader(final String key,
             final String value) {
@@ -271,12 +276,12 @@ public WrappedRequestBuilder withHeader(final String key,
             return this;
         }
 
-        /**
-         * Add a consumer that acts on a specific status code
-         *
-         * @param code             Status code
-         * @param responseConsumer Response consumer
-         * @return Builder instance
+        /*
+         Add a consumer that acts on a specific status code
+         
+         @param code             Status code
+         @param responseConsumer Response consumer
+         @return Builder instance
          */
         public WrappedRequestBuilder onStatus(final int code,
             final Consumer<HttpResponse> responseConsumer) {
@@ -284,11 +289,11 @@ public WrappedRequestBuilder onStatus(final int code,
             return this;
         }
 
-        /**
-         * Add a consumer that acts on all remaining status code
-         *
-         * @param responseConsumer Response consumer
-         * @return Builder instance
+        /*
+         Add a consumer that acts on all remaining status code
+         
+         @param responseConsumer Response consumer
+         @return Builder instance
          */
         public WrappedRequestBuilder onRemaining(
             final Consumer<HttpResponse> responseConsumer) {
@@ -296,11 +301,11 @@ public WrappedRequestBuilder onRemaining(
             return this;
         }
 
-        /**
-         * Add an exception consumer
-         *
-         * @param consumer Exception consumer
-         * @return Builder instance
+        /*
+         Add an exception consumer
+         
+         @param consumer Exception consumer
+         @return Builder instance
          */
         public WrappedRequestBuilder onException(
             final Consumer<Throwable> consumer) {
@@ -309,12 +314,12 @@ public WrappedRequestBuilder onException(
             return this;
         }
 
-        /**
-         * Perform the request
-         *
-         * @return The raw response, if no exception was thrown during the
-         *         handling of the response. If any exception was handled,
-         *         the method will return {@code null}
+        /*
+         Perform the request
+         
+         @return The raw response, if no exception was thrown during the
+                 handling of the response. If any exception was handled,
+                 the method will return {@code null}
          */
         public HttpResponse execute(int timeout) {
             for (final Consumer<WrappedRequestBuilder> decorator : settings.getRequestDecorators()) {
@@ -327,7 +332,8 @@ public HttpResponse execute(int timeout) {
                 }
                 final HttpResponse response = this.builder.build().executeRequest(timeout);
                 if (response != null) {
-                    final Consumer<HttpResponse> responseConsumer = this.consumers.getOrDefault(response.getStatusCode(), this.other);
+                    final Consumer<HttpResponse> responseConsumer =
+                        this.consumers.getOrDefault(response.getStatusCode(), this.other);
                     responseConsumer.accept(response);
                 }
                 if (throwables[0] != null) {
diff --git a/src/com/intellectualsites/http/HttpMethod.java b/src/com/intellectualsites/http/HttpMethod.java
index 3b92809..7ad7644 100644
--- a/src/com/intellectualsites/http/HttpMethod.java
+++ b/src/com/intellectualsites/http/HttpMethod.java
@@ -1,3 +1,7 @@
+/*
+  * #### This file has been modified by JxMake project #####
+  */
+
 /*
  * MIT License
  *
@@ -23,38 +27,38 @@
  */
 package com.intellectualsites.http;
 
-/**
- * HTTP methods
+/*
+ HTTP methods
  */
 enum HttpMethod {
 
-    /**
-     * Post requests are used to handle data
+    /*
+     Post requests are used to handle data
      */
     POST,
 
-    /**
-     * Get requests are handled for getting resources
+    /*
+     Get requests are handled for getting resources
      */
     GET,
 
-    /**
-     * Add data to an external source
+    /*
+     Add data to an external source
      */
     PUT,
 
-    /**
-     * Patch content on an external source
+    /*
+     Patch content on an external source
      */
     PATCH,
 
-    /**
-     * Retrieve the headers for a request
+    /*
+     Retrieve the headers for a request
      */
     HEAD(false),
 
-    /**
-     * Delete content from an external source
+    /*
+     Delete content from an external source
      */
     DELETE;
 
@@ -68,10 +72,10 @@ enum HttpMethod {
         this.hasBody = hasBody;
     }
 
-    /**
-     * Whether or not the method should return an entity
-     *
-     * @return Whether or not a response entity should be expected
+    /*
+     Whether or not the method should return an entity
+     
+     @return Whether or not a response entity should be expected
      */
     boolean hasBody() {
         return this.hasBody;
diff --git a/src/com/intellectualsites/http/HttpRequest.java b/src/com/intellectualsites/http/HttpRequest.java
index c5f8649..a86e511 100644
--- a/src/com/intellectualsites/http/HttpRequest.java
+++ b/src/com/intellectualsites/http/HttpRequest.java
@@ -1,3 +1,7 @@
+/*
+  * #### This file has been modified by JxMake project #####
+  */
+
 /*
  * MIT License
  *
@@ -36,9 +40,9 @@
 import java.util.function.Consumer;
 import java.util.function.Supplier;
 
-/**
- * HTTP request class. This should not be interacted with directly,
- * rather {@link com.intellectualsites.http.HttpClient} should be used
+/*
+ HTTP request class. This should not be interacted with directly,
+ rather {@link com.intellectualsites.http.HttpClient} should be used
  */
 final class HttpRequest {
 
@@ -62,10 +66,10 @@ private HttpRequest(final HttpMethod method, final URL url, final Headers header
         this.throwableConsumer = throwableConsumer;
     }
 
-    /**
-     * Create a new request {@link Builder builder}
-     *
-     * @return Builder instance
+    /*
+     Create a new request {@link Builder builder}
+     
+     @return Builder instance
      */
     static Builder newBuilder() {
         return new Builder();
@@ -102,7 +106,8 @@ HttpResponse executeRequest(int timeout) throws IOException {
                 final Object object = this.inputSupplier.get();
                 if (object != null) {
                     final EntityMapper.EntitySerializer serializer =
-                        this.mapper.getSerializer(object.getClass()).orElseThrow(() -> new IllegalArgumentException(String
+                        this.mapper.getSerializer(object.getClass())
+                            .orElseThrow(() -> new IllegalArgumentException(String
                             .format("There is no registered serializer for type '%s'",
                                 object.getClass().getCanonicalName())));
                     if (this.headers.getHeader("Content-Type").isEmpty()) {
@@ -177,67 +182,69 @@ static final class Builder {
         private Builder() {
         }
 
-        /**
-         * Specify the entity mapper used by the request
-         *
-         * @param mapper Entity mapper
-         * @return Builder instance
+        /*
+         Specify the entity mapper used by the request
+         
+         @param mapper Entity mapper
+         @return Builder instance
          */
         Builder withMapper(final EntityMapper mapper) {
             this.mapper = Objects.requireNonNull(mapper, "Mapper may not be null");
             return this;
         }
 
-        /**
-         * Specify the HTTP method used in the request
-         *
-         * @param method HTTP method
-         * @return Builder instance
+        /*
+         Specify the HTTP method used in the request
+         
+         @param method HTTP method
+         @return Builder instance
          */
         Builder withMethod(final HttpMethod method) {
             this.method = Objects.requireNonNull(method, "Method may not be null");
             return this;
         }
 
-        /**
-         * Specify the URL used in the request
-         *
-         * @param url URL
-         * @return Builder instance
+        /*
+         Specify the URL used in the request
+         
+         @param url URL
+         @return Builder instance
          */
         Builder withURL(final URL url) {
             this.url = Objects.requireNonNull(url, "URL may not be null");
             return this;
         }
 
-        /**
-         * Add a header to the request
-         *
-         * @param key Header key
-         * @param value Header value
-         * @return Builder instance
+        /*
+         Add a header to the request
+         
+         @param key Header key
+         @param value Header value
+         @return Builder instance
          */
         Builder withHeader(final String key, final String value) {
-            this.headers.addHeader(Objects.requireNonNull(key, "Key may not be null"), Objects.requireNonNull(value, "Value may not be null"));
+            this.headers.addHeader(
+                Objects.requireNonNull(key, "Key may not be null"),
+                Objects.requireNonNull(value, "Value may not be null"));
             return this;
         }
 
-        /**
-         * Add an input entity to the request
-         *
-         * @param inputSupplier Input supplier
-         * @return Builder instance
+        /*
+         Add an input entity to the request
+         
+         @param inputSupplier Input supplier
+         @return Builder instance
          */
         Builder withInput(final Supplier<Object> inputSupplier) {
             this.inputSupplier = Objects.requireNonNull(inputSupplier, "Input supplier may not be null");
             return this;
         }
 
-        /**
-         * Add a throwable consumer
-         *
-         * @param consumer Throwable consumer
-         * @return Builder instance
+        /*
+         Add a throwable consumer
+         
+         @param consumer Throwable consumer
+         @return Builder instance
          */
         Builder onException(final Consumer<Throwable> consumer) {
             this.throwableConsumer = Objects.requireNonNull(consumer, "Consumer may not be null");
diff --git a/src/com/intellectualsites/http/HttpResponse.java b/src/com/intellectualsites/http/HttpResponse.java
index d61b0c0..038f2e1 100644
--- a/src/com/intellectualsites/http/HttpResponse.java
+++ b/src/com/intellectualsites/http/HttpResponse.java
@@ -1,3 +1,7 @@
+/*
+  * #### This file has been modified by JxMake project #####
+  */
+
 /*
  * MIT License
  *
@@ -25,8 +29,8 @@
 
 import java.util.Objects;
 
-/**
- * A HTTP response
+/*
+ A HTTP response
  */
 public final class HttpResponse {
 
@@ -48,58 +52,58 @@ private HttpResponse(final int code,
         this.body = body;
     }
 
-    /**
-     * Create a new builder instance
-     *
-     * @return Builder instance
+    /*
+     Create a new builder instance
+     
+     @return Builder instance
      */
     static Builder builder() {
         return new Builder();
     }
 
-    /**
-     * Get the HTTP status message
-     *
-     * @return Status message
+    /*
+     Get the HTTP status message
+     
+     @return Status message
      */
     public String getStatus() {
         return this.status;
     }
 
-    /**
-     * Get the HTTP status code
-     *
-     * @return Status code
+    /*
+     Get the HTTP status code
+     
+     @return Status code
      */
     public int getStatusCode() {
         return this.code;
     }
 
-    /**
-     * Get the raw response body
-     *
-     * @return Response body
+    /*
+     Get the raw response body
+     
+     @return Response body
      */
     public byte[] getRawResponse() {
         return this.body;
     }
 
-    /**
-     * Get the response headers
-     *
-     * @return Response headers
+    /*
+     Get the response headers
+     
+     @return Response headers
      */
     public Headers getHeaders() {
         return this.headers;
     }
 
-    /**
-     * Get the response entity and map it to a specific type
-     *
-     * @param returnType Return type class
-     * @param <T> Return type
-     * @return Response
-     * @throws IllegalArgumentException If no mapper exists for the type
+    /*
+     Get the response entity and map it to a specific type
+     
+     @param returnType Return type class
+     @param <T> Return type
+     @return Response
+     @throws IllegalArgumentException If no mapper exists for the type
      */
     public <T> T getResponseEntity(final Class<T> returnType) {
         final String contentTypeString = this.headers.getOrDefault("content-type", null);
diff --git a/src/com/intellectualsites/http/LICENSE.txt b/src/com/intellectualsites/http/LICENSE.txt
index bf9298f..d890e2d 100644
--- a/src/com/intellectualsites/http/LICENSE.txt
+++ b/src/com/intellectualsites/http/LICENSE.txt
@@ -1,3 +1,7 @@
+/*
+  * #### This file has been modified by JxMake project #####
+  */
+
 /*
  * MIT License
  *
