diff -ru ATmega8A_Boost.ORIG/boost.h ATmega8A_Boost/boost.h
--- ATmega8A_Boost.ORIG/boost.h	2026-01-03 07:42:36.000000000 +0700
+++ ATmega8A_Boost/boost.h	2026-04-27 07:38:23.160702113 +0700
@@ -124,7 +124,7 @@
 		// Save current error for next iteration
 		prevError = error;
 
-		// Adjust the duty cyle PWM
+		// Adjust the duty cycle PWM
 		int16_t duty = ( (int16_t) pwmGet() ) + (int16_t) (PID / 4);
 
 		     if(duty < PWM_DUTY_MIN) duty = PWM_DUTY_MIN;
diff -ru ATmega8A_Boost.ORIG/main.c ATmega8A_Boost/main.c
--- ATmega8A_Boost.ORIG/main.c	2026-01-03 07:42:36.000000000 +0700
+++ ATmega8A_Boost/main.c	2026-04-27 07:44:22.495744529 +0700
@@ -129,6 +129,7 @@
 		if( gget_nRST() ) {
 			printf_P( PSTR(">>> Reset signal - begin (ADC_Ref = %d)\n"), boostVoltage );
 			enaVppVdd();
+			// Wait for nRST to go high, resetting watchdog without delay
 			while( gget_nRST() ) delayMS(0);
 			disVppVdd();
 			printf_P( PSTR(">>> Reset signal - end\n\n") );
