diff -ru ATxmega128A4U.ORIG/JxMakeUSBGPPIO.c ATxmega128A4U/JxMakeUSBGPPIO.c
--- ATxmega128A4U.ORIG/JxMakeUSBGPPIO.c	2026-01-03 07:42:36.000000000 +0700
+++ ATxmega128A4U/JxMakeUSBGPPIO.c	2026-05-22 04:31:11.251294029 +0700
@@ -439,17 +439,17 @@
 			case CDC_PARITY_Space: hwuxrt_deferSetParityMode(UXRT_Parity_None);
 			                       CDCInterfaceInfo->State.LineEncoding.ParityType = CDC_PARITY_None;
 			                       break;
-		} // case
+		} // switch
 
 		// Handle number of stop bits
 		switch(CDCInterfaceInfo->State.LineEncoding.CharFormat) {
-			case CDC_LINEENCODING_OneStopBit          : hwuxrt_deferSetNumStopBit(UXRT_StopBit_1); break;
+			case CDC_LINEENCODING_OneStopBit          : hwuxrt_deferSetNumStopBit(UXRT_StopBit_1);
 			                                            break;
 			case CDC_LINEENCODING_OneAndAHalfStopBits : /* FALLTHROUGH */
 			case CDC_LINEENCODING_TwoStopBits         : hwuxrt_deferSetNumStopBit(UXRT_StopBit_2);
 			                                            CDCInterfaceInfo->State.LineEncoding.CharFormat = CDC_LINEENCODING_TwoStopBits;
 			                                            break;
-		} // case
+		} // switch
 
 		// Handle number of data bits
 		switch(CDCInterfaceInfo->State.LineEncoding.DataBits) {
@@ -463,7 +463,7 @@
 			case 9 : hwuxrt_deferSetCharacterSize(UXRT_CharSize_8);
 			         CDCInterfaceInfo->State.LineEncoding.DataBits = 8;
 			         break;
-		} // case
+		} // switch
 
 		// Apply the new configuration if the UART/USRT is enabled and JTAG is not enabled
 		if( hwuxrt_isEnabled() && !jtag_isEnabled() ) {
diff -ru ATxmega128A4U.ORIG/Modules/JTAG.c ATxmega128A4U/Modules/JTAG.c
--- ATxmega128A4U.ORIG/Modules/JTAG.c	2026-01-03 07:42:36.000000000 +0700
+++ ATxmega128A4U/Modules/JTAG.c	2026-04-27 07:13:55.249686771 +0700
@@ -106,7 +106,7 @@
 		*data = *data | _jtag_tdo_7(); // Read and store the TDO bit that has been received (LSB first)
 		_jtag_tck(false);              // Make the TCK inactive
 
-	} // false
+	} // for
 }
 
 
@@ -333,7 +333,7 @@
 	if(drShift && irShift) return false;
 
 	// Simply exit if the number of pairs is zero
-	if(nPairs) return true;
+	if(!nPairs) return true;
 
 	// Perform state transition as needed
 	if(drShift || irShift) {
