diff -ru ATmega8A_UPDI.ORIG/updi.h ATmega8A_UPDI/updi.h
--- ATmega8A_UPDI.ORIG/updi.h	2026-01-03 07:42:36.000000000 +0700
+++ ATmega8A_UPDI/updi.h	2026-04-27 07:38:23.161702111 +0700
@@ -155,7 +155,7 @@
 
 	gdis_UPDI_KEY();              // Disable the software UART pin (this will allow the upstream UPDI signal from the programmer to reconnect to the target)
 
-	wdt_reset(); printf_P( PSTR("... updiBegin() - enabling INT0 by storing the inital the tick value\n") );
+	wdt_reset(); printf_P( PSTR("... updiBegin() - enabling INT0 by storing the initial tick value\n") );
 
 	tickGetTo(&_updi_tick);       // Save the initial UPDI tick
 
